### PR TITLE
Allow empty user to be used

### PR DESCRIPTION
**Pull request by glatzert**
Amends #21 which was described in #20 

These changes should allow one to set the username to an empty value.

### CODE REVIEW

File: src/Tasks/InstallWindowsScheduledTask/Code/Shared/UserInputToScheduledTaskMapper.psm1 [hunk 1]
```diff
@@ -19,7 +19,7 @@ function Get-AccountCredentialsToRunScheduledTaskAs
 {
```
Comment (deadlydog) [inline src/Tasks/InstallWindowsScheduledTask/Code/Shared/UserInputToScheduledTaskMapper.psm1:19]: Can you add these changes to `src\SharedTaskCode\UserInputToScheduledTaskMapper.psm1` and run `src\SharedTaskCode\_CopyModuleFilesToTasksSharedCodeDirectories.ps1` to copy the changes to all the tasks.

**Comment by deadlydog** [comment]
Thanks!

**Comment by glatzert** [comment]
Hi there, I'd like to ask, if you'd be willing to deploy the version containing this code.

**Comment by deadlydog** [comment]
Sorry, I thought I had deployed this new version out already. It's now available in version 2.3.12.

**Pull request merged**